back/app/models/custom_field.rb

```diff
@@ -23,6 +23,9 @@
 #  maximum_label_multiloc :jsonb            not null
 #  logic                  :jsonb            not null
 #  answer_visible_to      :string
+#  select_count_enabled   :boolean          default(FALSE), not null
```

Comment:
I think we need to add some validations here:
- select_count_enabled cannot be nil (otherwise we will get a 500 server error because of the database error)
- maximum_select_count and minimum_select_count can probably not be smaller than zero and probably cannot be more than the number of options.

Reply:
I was able to add the validation for select_count_enabled, but I'm not quite sure how to add a validation for the options count here, since it isn't part of this table :thinking: @jamesspeake Do you know of an example in the code base where validation uses values stored in another table which I could reference? I'm having some trouble finding one.